demo-project/conf/base/catalog_01_raw.yml

```diff
@@ -1,14 +1,38 @@
 companies:
   type: pandas.CSVDataSet
   filepath: ${base_location}/01_raw/companies.csv
-  layer: raw
+  metadata:
+    kedro-viz:
+      layer: raw
 
 reviews:
   type: pandas.CSVDataSet
   filepath: ${base_location}/01_raw/reviews.csv
-  layer: raw
+  metadata:
+    kedro-viz:
+      layer: raw
 
 shuttles:
   type: pandas.ExcelDataSet
   filepath: ${base_location}/01_raw/shuttles.xlsx
-  layer: raw
+  metadata:
+    kedro-viz:
+      layer: raw
+
+
+
+
+# companies:
```

Comment:
I have commented the old way of doing things in the catalog.yml files so you can test both using layers as a top-level attribute or inside metadata. I will remove the comments before merge.